pkg/common/common.go

```diff
@@ -21,7 +21,7 @@ const (
 	grafanaAgentResourceName = "grafana-agent-secret"
 )
 
-func GetGrafanaAgentResourceName() string {
+func GrafanaAgentResourceName() string {
```

Comment:
```suggestion
func GrafanaAgentResourceName() string {
```

That's still a secret, not a resource

Reply:
IIRC, when I set GrafanaAgentSecretName, I had validation issues because of `secret` in the name

Reply:
Sure but we can ignore that using something like https://github.com/giantswarm/logging-operator/blob/7715f96f71bfed748b5ff465e59495a6acf36c6c/pkg/resource/loki-auth/loki-auth.go#L22